File: 2021/template.py
import os
import sys


# get data path
file_name = os.path.basename(__file__).split('.')[0]
infile = sys.argv[1] if len(sys.argv)>1 else f'input/{file_name}.txt'

with open(infile) as f:
    data = f.readlines()

print(data)

File: 2021/template.js

import { promises as fs } from 'fs'


const parseInput = async () => {
  let data = await fs.readFile('input/day_8.txt', 'utf-8')
  return data
}

const part1 = async () => {
  const data = await parseInput()
	console.log("asdf")
}

part1()
